route.go

```diff
@@ -261,6 +261,7 @@ func (r *Route) create(path []string) *Route {
 
 	// set the pattern name
 	newRoute.pattern = path[1]
+	newRoute.fullPath = r.fullPath + "/" + path[1]
```

Comment:
Actually, invalid benchmark, the first one is doing an extra join.  Looks like the manual concat is still better then anything else??? Not sure why.